client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -135,6 +135,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			this.props.updateFields( { [ name ]: ! this.props.fields[ name ] } );
 		};
 
+		handleAutosavingToggle = name => () => {
```

Comment:
ahh probably, yes... What do you think about refactoring `handlToggle` to have this signature `handleToggle( name, autoSaving=false )` and moving the behavior here instead?